queries.sql

```diff
@@ -0,0 +1,133 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE users SET nickname = 'test' 
+WHERE id = 1;
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT * FROM products 
+WHERE user_id = 1 ORDER BY created_at DESC 
+LIMIT 10 
+OFFSET 20;
+
+
+/*
+  3. 내가 생성한 상품의 총 개수
+*/
+SELECT COUNT(*) AS total_products
+FROM products
+WHERE user_id = 1;
+
+/*
+  4. 내가 좋아요 누른 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT products.*
+FROM product_likes
+	JOIN products ON products.id = product_likes.product_id
+WHERE product_likes.user_id = 1
+GROUP BY products.name, products.id
```

Comment:
```suggestion
GROUP BY products.id
```
로만 묶어도 동일한 결과를 가져옵니다 : )